README.md

```diff
@@ -96,12 +99,8 @@ metadata:
 spec:
   description: string # optional
   service: [service name] # name of the service to associate this SLO with
-  indicator: # represents the Service Level Indicator (SLI)
-    thresholdMetric: # represents the metric used to inform the Service Level Object in the objectives stanza
-      source: string # data source for the metric
-      queryType: string # a name for the type of query to run on the data source
-      query: string # the query to run to return the metric
-      metadata: # optional, allows data source specific details to be passed
+  indicator: # see SLI below for details
```

Comment:
Just so I'm clear on this, we can either specific the `indicator` "inline" here as before, or provide an `indicatorRef` to an SLI defined somewhere else?

Reply:
Yes.